src/panels/lovelace/special-rows/hui-attribute-row.ts

```diff
@@ -72,7 +73,7 @@ class HuiAttributeRow extends LitElement implements LovelaceRow {
               ></hui-timestamp-display>`
             : typeof attribute === "number"
             ? formatNumber(attribute, this.hass.locale)
-            : attribute ?? "-"}
+            : formatAttributeValue(this.hass, attribute ?? "-")}
```

Comment:
No need to run those regex on the "-"
```suggestion
            : attribute ? formatAttributeValue(this.hass, attribute) : "-"}
```

Reply:
This should be changed in the other places too.